docs/contributor_guide/index.rst

```diff
@@ -18,3 +18,69 @@ The contribution guide explains how to structure your contributions. Please
    contributing_code
    contributing_example_notebooks
    fairlearn_proposals
+
+.. _how_to_talk_about_fairness:
+
+How to talk and/or write about fairness in AI
+---------------------------------------------
+
+- Be clear that there is no single definition of fairness that will apply
```

Comment:
This is a rather long bulleted list, and I think it should be split up a bit. Preferably with a sentence or two of 'normal' text in between the sections of bullets.

I suggest starting with a sentence or two of introduction, then putting the "There are many reasons why AI systems behave unfairly...", "be specific about various types of harm" etc. points. Then, have another sentence or two of regular text along the lines of "These are the points you need to make clearly" and list your "Be clear" points (dropping the "Be clear" prefix from each).

Reply:
So this is for contributors, to give them an idea on how to write/talk about fairness. The overall section on fairness in AI is in the user guide. Perhaps they need to be in the same place, or perhaps we don't mind duplication.
@MiroDudik  @hannawallach FYI

I'm certainly on board with making this more readable. Will take a stab.

Reply:
I wasn't meaning to change the purpose of this at all - just that a bare list of bullet points might not be easiest to read.

Reply:
I understand. Just providing some more context :-)

Reply:
This has all been rewritten now. Resolving the old comments because they refer to a completely different version. Let me know if you think this is better. I can always go back to the original.